.gitignore

```diff
@@ -2,6 +2,7 @@
 .vscode/
 .DS_Store
 **/*/.idea/
+/build/
```

Comment:
Whats this for?

Reply:
@KyroVibe This is just for if you want to build the project into a build folder. I we have this inside of `engine/` I just added it to the root of the project.